From 22ab58e8f958980c070a46cabe197b3557dda0a3 Mon Sep 17 00:00:00 2001
From: Matthias Reichl <hias@horus.com>
Date: Sun, 6 Sep 2020 23:50:53 +0200
Subject: [PATCH] [cmake] support libplist version 2.2.0

libplist 2.2.0 changed the pkgconfig and library names from
libplist(++) to libplist(++)-2.0. Add these names to cmake pkgconfig
and library lists so cmake can pick up external 2.2 versions as well.

Signed-off-by: Matthias Reichl <hias@horus.com>
---
 cmake/modules/FindPlist.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/modules/FindPlist.cmake b/cmake/modules/FindPlist.cmake
index 2c86b7493a..8f9b2d6cd7 100644
--- a/cmake/modules/FindPlist.cmake
+++ b/cmake/modules/FindPlist.cmake
@@ -15,7 +15,7 @@
 #   Plist::Plist   - The Plist library
 
 if(PKG_CONFIG_FOUND)
-  pkg_check_modules(PC_PLIST libplist QUIET)
+  pkg_search_module(PC_PLIST libplist-2.0 libplist QUIET)
 endif()
 
 find_path(PLIST_INCLUDE_DIR plist/plist.h
@@ -23,7 +23,7 @@ find_path(PLIST_INCLUDE_DIR plist/plist.h
 
 set(PLIST_VERSION ${PC_PLIST_VERSION})
 
-find_library(PLIST_LIBRARY NAMES plist libplist
+find_library(PLIST_LIBRARY NAMES plist-2.0 plist libplist-2.0 libplist
                                  PATHS ${PC_PLIST_LIBDIR})
 
 include(FindPackageHandleStandardArgs)
-- 
2.20.1

